### PR TITLE
Expose order params

### LINKED ISSUE #1255: Expose order params for World__ModelConnection
**Is your feature request related to a problem? Please describe.**
I'd like to order models by name in alphabetical order.

**Describe the solution you'd like**

```graphql
query Models {
  models(order: { field: NAME }) {
    edges {
      node {
        name
      }
    }
  }
}
```

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by gianalarcon**
Closes #1255
 - Implement `ModelObject` struct. Create fn `new()` for ModelObject
 - No `direction` input value for World__ModelOrder
 - Implement fn `resolve_many()` for `ModelObject`
 
 @broody

### CODE REVIEW

File: crates/torii/graphql/src/object/connection/mod.rs [hunk 1]
```diff
@@ -123,9 +123,16 @@ pub fn connection_output(
...
+                Some(order) => {
+                    if is_external {
+                        format!("external_{}", order.field)
+                    } else {
+                        order.field.to_string()
+                    }
+                }
                 None => id_column.to_string(),
             };
+            println!("connection_output order_field {}", order_field);
```
Comment (broody) [inline crates/torii/graphql/src/object/connection/mod.rs:135]: Let's remove all the debug prints, there are a couple

File: crates/torii/graphql/src/tests/models_ordering_test.rs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        );
+
+        let result = run_graphql_query(schema, &query).await;
+        result.get("models").ok_or("models not found").unwrap().clone()
+    }
+
+    // End to end test spins up a test sequencer and deploys types-test project, this takes a while
+    // to run so combine all related tests into one
+    #[tokio::test(flavor = "multi_thread")]
+    async fn models_test() -> Result<()> {
```
Comment (broody) [inline crates/torii/graphql/src/tests/models_ordering_test.rs:46]: Let's rename to models_ordering_test

**Review by broody — COMMENTED** [review]
Looks good, just some minor things

File: crates/torii/graphql/src/tests/models_ordering_test.rs (hunk 2 above)
Comment (gianalarcon) [reply crates/torii/graphql/src/tests/models_ordering_test.rs]: Fixed on commit 2cc3c38

**Review by broody — APPROVED** [review]

**Pull request merged**